ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/TransactionFactory.java

```diff
@@ -79,9 +79,8 @@ private Transaction createEeaTransaction(final JsonRpcRequest request) {
 
     if (params.privacyGroupId().isPresent()) {
       final NonceProvider nonceProvider =
-          new PantheonPrivateNonceProvider(
-              pantheon, params.sender(), params.privacyGroupId().get());
-      return PantheonPrivateTransaction.from(params, nonceProvider, request.getId());
+          new BesuPrivateNonceProvider(pantheon, params.sender(), params.privacyGroupId().get());
```

Comment:
can we get rid of this patheon as well?

Reply:
I was leaving that one until the library is updated